.github/CODEOWNERS

```diff
@@ -20,7 +20,6 @@ dreamchecker.exe @AffectedArc07
 rust_g.dll @AffectedArc07
 
 
-
```

Comment:
webeditor heresy caused this, and and the best bit is you don't have to care anymore